src/lessons/14_form_validation_recap_dom/form-validation-recap-dom.slides.html

```diff
@@ -0,0 +1,416 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>Form Validation + Recap Dom manipulations</title>
+  <meta name="date" content="2021-04-26" />
+  <meta name="description" content="
+      - Recap of DOM manipulation
+      - Form validation
+    " />
+</head>
+
+<body>
+  <div class="reveal">
+    <div class="slides" id="javascript-recap-dom">
+      <section>
+        <p>Hey! 👋🏼</p>
+        <h1 class="r-fit-text">Form Validation + Recap Dom manipulations</h1>
+        <p>Redi School Munich - Spring 2021</p>
+      </section>
+
+      <section class="initial">
+        <section>
+          <h2>Recap on DOM Manipulation</h2>
+          <a href="https://github.com/alena456">Kahoot!</a>
```

Comment:
Is this supposed to be your github profile? 👀

Reply:
I still have to update the Link, thanks for the hint!